<?xml version="1.0" encoding="UTF-8"?>
<_ui version="4.0">
    <class>animation</class>
    <widget class="QWidget" name="animation">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>576</width>
                <height>425</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Advanced</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="4">
                <layout class="QGridLayout" name="mainLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="radius">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>71</width>
                                    <height>27</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>radius:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <widget class="QSpinBox" name="spinBox">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>71</width>
                                    <height>27</height>
                                </size>
                            </property>
                            <property name="minimum">
                                <number>1</number>
                            </property>
                            <property name="maximum">
                                <number>9</number>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="0">
                        <widget class="QLabel" name="points">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>points:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="3" column="1">
                        <widget class="QLabel" name="Y">
                            <property name="minimumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>Y:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="3" column="0">
                        <widget class="QLabel" name="X">
                            <property name="minimumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>71</width>
                                    <height>26</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>X:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QTextEdit" name="editRadius">
                            <property name="enabled">
                                <bool>true</bool>
                            </property>
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>71</width>
                                    <height>27</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>71</width>
                                    <height>27</height>
                                </size>
                            </property>
                            <property name="placeholderText">
                                <string>1</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="rotation">
                            <property name="text">
                                <string>rotation:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QTextEdit" name="editRotation">
                            <property name="maximumSize">
                                <size>
                                    <width>71</width>
                                    <height>27</height>
                                </size>
                            </property>
                            <property name="html">
                                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
                                    &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style
                                    type=&quot;text/css&quot;&gt;
                                    p, li { white-space: pre-wrap; }
                                    &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu';
                                    font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
                                    &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px;
                                    margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot;
                                    font-family:'Ubuntu';&quot;&gt; &lt;/span&gt;&lt;/p&gt;
                                    &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px;
                                    margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                </string>
                            </property>
                            <property name="placeholderText">
                                <string>0</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="2" column="4">
                <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>150</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="1" column="0">
                <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>40</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="1" column="1">
                <spacer name="horizontalSpacer">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>40</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="4" column="4">
                <widget class="QPushButton" name="buttonPresets">
                    <property name="text">
                        <string>presets</string>
                    </property>
                </widget>
            </item>
            <item row="3" column="4">
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>10</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="1" column="3">
                <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>40</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="1" column="2">
                <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>40</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</_ui>
